### PR TITLE
Add English about page

### PR DESCRIPTION
**Pull request by MattiSG**

### CODE REVIEW

File: en/apropos.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
```
Comment (Morendil) [inline en/apropos.md]: en/about pour le nom du fichier ?

File: en/apropos.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+title: About
+layout: text
+---
+
+## Background
+
+beta.gouv.fr is the Digital Services Incubator, a mission of the French government’s Secretariat-general for Government Modernisation. Our aim is to spread the culture of digital innovation throughout the administration. We do this through State Startups.
+
+A State Startup is not just an oxymoron, but a match between a team and a mission. There is no capital investment, no separate juridical entity, only the usage of the tools and practices of the digital startups to solve a friction in an interaction an administration has with citizens. We help it identify an “intrapreneur”, one of its civil servants ready to hustle her way into delivering a digital public service, and create a team of experts around her. We hire developers, datascientists, designers, product managers and all needed talents, either under short-term contracts or as independent contractors, and coach that team so it stays laser-focused on solving real-world issues for real-world users. This team of 2 to 4 people will have 6 months to prove a digital product can improve the situation.
```
Comment (Morendil) [inline en/apropos.md]: "Not just" implies it _is_ an oxymoron, which may not be what you want to assert?

File: en/apropos.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+## Objectives and main results achieved
+
+The new version of the French Open Data Portal (data.gouv.fr) was the first State Startup to come into existence, back in June of 2013. Made publicly available in December that same year, it brought social features to a tool that used to be a catalog of spreadsheets —as in every other country at the time. The new interface created feedback loops from civil society, that could now publish reuses, towards the institutions producing data, thus allowing them to increase the quality of their datasets by leveraging the crowd. Being open-source, this tool is now used in other countries such as Luxembourg, transferring capabilities EU-wide.
```
Comment (Morendil) [inline en/apropos.md]: Missing space after dash ("- as in every other...")

File: en/apropos.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+Our continued growth, our now 27 products, as well as our increasing number of partner ministries, all prove that building digital services for and with the public, ensuring collaboration across government agencies and beyond, ignoring usual committee-based, siloed decision-making, yields greatly improved services at a fraction of the cost and fosters interest and support from the civil society.
+
+## Lessons learnt
+
+We have learnt that the best strategy in digital transformation is delivery. Rather than trying to convince other stakeholders —be they fellow administrations, the private sector or individuals— to work towards a specific goal, it is more efficient to start working on it, while opening at the same time as many ways for contribution as possible. As feedback is truly listened to and taken action upon, a community starts shaping itself and the power of the crowd is leveraged. That support can then be shifted into resources, but also into political power to support larger transformation efforts, convincing more reluctant actors to consider digital transformation as an opportunity rather than a challenge.
```
Comment (Morendil) [inline en/apropos.md]: Missing (or extra) spaces around dashes - according to [this](http://www.thepunctuationguide.com/em-dash.html), but perhaps you are following a different reference, in which case feel free to ignore this remark. Like above, this just looks a bit odd to my eye.

File: en/apropos.md (hunk 1 above)
Comment (MattiSG) [inline en/apropos.md]: J'ai hésité, mais cela permet de simplement préfixer une URL francophone de `/en/` pour avoir la version anglaise.

Le mieux serait probablement de nommer `about` et de mettre un `redirect_from`.

File: en/apropos.md (hunk 2 above)
Comment (MattiSG) [inline en/apropos.md]: It is exactly what I want to assert.

File: en/apropos.md (hunk 3 above)
Comment (MattiSG) [inline en/apropos.md]: No, this is an em dash and in English. Punctuation is disputed, but it is quite often set close.

[Read more](https://en.wikipedia.org/wiki/Dash#Spacing_and_substitution).

File: en/apropos.md (hunk 4 above)
Comment (MattiSG) [inline en/apropos.md]: Typesetting of an em-dash looks different in a non-monospace font. I'm ok for removing the external spaces if you care, I'm used to following setting close (i.e. no space) as a typographic best practice, compensated by screenreader compatibility which makes me add external spaces.

**Pull request merged**